test/test-lib/api-helpers.ts

```diff
@@ -1,8 +1,9 @@
+import type { PinejsClient } from '@balena/pinejs/out/sbvr-api/sbvr-utils';
```

Comment:
@thgreasi can you remove the deep imports like this please

Reply:
OOOoppps, sorry for that.
See: https://github.com/balena-io/open-balena-api/pull/1500